org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/wizard/welcome/GradleWelcomeWizardPage.java

```diff
@@ -9,12 +9,17 @@
  *     Etienne Studer & Donát Csikós (Gradle Inc.) - initial API and implementation and initial documentation
  */
 
-package org.eclipse.buildship.ui.wizard.project;
+package org.eclipse.buildship.ui.wizard.welcome;
 
-import com.google.common.collect.ImmutableList;
-
-import com.gradleware.tooling.toolingutils.binding.Property;
+import java.util.Collection;
 
+import org.eclipse.buildship.core.GradlePluginsRuntimeException;
+import org.eclipse.buildship.core.projectimport.ProjectImportConfiguration;
+import org.eclipse.buildship.ui.UiPlugin;
+import org.eclipse.buildship.ui.wizard.project.AbstractProjectWizard;
+import org.eclipse.buildship.ui.wizard.project.AbstractWizardPage;
```

Comment:
Having references from the 'welcome' package to the 'project' package seems wrong to me. It actually leads to a package cycle. I do not see the need for the welcome package.